website/scripts/ros/ros_host_vehicle_cmd.js

```diff
@@ -126,4 +156,42 @@ function subscribeToSteeringWheel()
             updateSteeringWheel(steer_percentage+ "%",rotateDegreeModule);
         }  
     });
+}
+
+
+/**steering_feedback from ssc_interface package
+ * Topic: /hardware_interface/steering_feedback 
+ * Message: automotive_platform_msgs/BrakeFeedback
+            std_msgs/Header header
+                uint32 seq
+                time stamp
+                string frame_id
+            float32 steering_wheel_angle
+ */
+function sunscribeToSteeringFeedback()
+{
+    var listener = new ROSLIB.Topic({
+        ros:g_ros,
+        name: T_STEERING_WHEEL_FEEDBACK,
+        messageType: M_STEERING_FEEDBACK
+    });
+    listener.subscribe(function(message){
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+
+        if(message!=null && message.steering_wheel_angle!=null)
+        {
+            let value, vehicle_steer_ratio,vehicle_steer_lim_deg = '';
+            vehicle_steer_lim_deg = session_hostVehicle.steeringLimit;
+            vehicle_steer_ratio =session_hostVehicle.steeringRatio;
+            value = message.steering_wheel_angle;
+
+            let rotateDegree = Math.ceil(value * 180/(Math.PI));
```

Comment:
@dan-du-car I spoke to Mike and it seems that it can be more straightforward if you simplify it to this one liner. 

steering percentage = current_steering_angle / (vehicle_steering_gear_ratio * vehicle_steer_lim_deg * DEG2RAD) 

Where DEG2RAD is 0.0174533. This is more straight forward than what you have now. It does arrive to the same value based on calculation on excel. I prefer you make it more straighforward to understand.

Reply:
It is addressed